zapisy/apps/enrollment/courses/templates/courses/course_parts/course_head.html

```diff
@@ -16,6 +16,22 @@ <h1 class="d-inline-block">
 </div>
 
 <table class="table table-bordered table-md-responsive" id="table-info">
+    <style>
+        #table-info ul {
+            list-style-type: none;
+            padding: 0;
+            margin: 0;
+        }
+
+        #hours ul li {
```

Comment:
Proszę jeszcze o eksperyment z taką zmianą (o ile nie ma jakiegoś ważnego powodu, żeby zostawić tak, jak jest):
```suggestion
        #table-info ul li {
```
– to nie ma znaczenia dla bieżącego efektu (ten już jest OK) ale byłoby to zgodne z intencją, jak widzimy **wszystkie** listy wypunktowane, jakie zdarzają się w tych tabelkach (choć inne już mają podobny wygląd dzięki dziedziczonemu po Bootstrapowej klasie `badge` własności `display: inline-block`).